Dockerfile

```
@@ -23,8 +23,9 @@ RUN addgroup -S ${GROUP} -g ${GID} && adduser -D -S -u ${UID} ${USER} ${GROUP} &
 
 COPY --from=build-stage --chown=${USER}:${GROUP} /app/dist /www/
 COPY --chown=${USER}:${GROUP} entrypoint.sh /entrypoint.sh
+RUN mv /www/assets /www/default-assets
```

We can't assume that **/www/assets** will be accessible to the UID that is running the build. On my system I get **access denied**

```suggestion
Replace with `COPY --from=build-stage --chown=${USER}:${GROUP} /app/dist/assets /www/default-assets`